tests/test_cli.py

```diff
@@ -1,53 +1,37 @@
-from unittest import mock
-from unittest.mock import mock_open
-
-from rapids_dependency_file_generator.cli import generate_file_obj, generate_matrix
-
-mock_yaml = """
-files:
-  test:
-    output: none
-    includes:
-      - build
-      - test
-"""
-
-
-@mock.patch(
-    "builtins.open",
-    mock_open(read_data=mock_yaml),
-)
-def test_generate_file_obj():
-    # valid env
-    file_obj = generate_file_obj(
-        "na_file", "test", "conda", {"cuda": ["11.5"], "arch": ["x86_64"]}
-    )
-
-    assert file_obj == {
-        "test": {
-            "output": "conda",
-            "matrix": {"cuda": ["11.5"], "arch": ["x86_64"]},
-            "includes": ["build", "test"],
-        }
-    }
-
-    # invalid env: missing config_file
-    file_obj = generate_file_obj("", "file_key", "file_type", "matrix")
-    assert file_obj == {}
-
-    # invalid env: missing file_key
-    file_obj = generate_file_obj("config_file", "", "file_type", "matrix")
-    assert file_obj == {}
+import pytest
 
-    # invalid env: missing file_type
-    file_obj = generate_file_obj("config_file", "file_key", "", "matrix")
-    assert file_obj == {}
-
-    # invalid env: missing matrix
-    file_obj = generate_file_obj("config_file", "file_key", "file_type", "")
-    assert file_obj == {}
+from rapids_dependency_file_generator.cli import generate_matrix, validate_args
 
 
 def test_generate_matrix():
     matrix = generate_matrix("cuda=11.5;arch=x86_64")
     assert matrix == {"cuda": ["11.5"], "arch": ["x86_64"]}
+
+    matrix = generate_matrix(None)
+    assert matrix == {}
+
+
+def test_validate_args():
+    # Missing output
+    with pytest.raises(Exception):
```

Comment:
Try to use a specific `Exception` subclass like `ValueError`. It's a more specific test and will prevent this from catching `KeyboardInterrupt` like `Ctrl+C` / `^C`.